### PR TITLE
Fix copying a question using the default course

### PR DESCRIPTION
**Pull request by nwalters512**
This handles the case where someone tries to copy a question to the default (first) option in the course `<select>` in the question copy modal. If the selected option didn't change, our little script would never have set the appropriate action or CSRF token. Now, they default to the values corresponding to the first course in the options.

### CODE REVIEW

File: apps/prairielearn/src/pages/partials/question.ejs [hunk 1]
```diff
@@ -218,7 +218,7 @@
   <div id="copyQuestionModal" class="modal fade" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
     <div class="modal-dialog">
       <div class="modal-content">
-        <form method="POST">
+        <form method="POST" action="<%= question_copy_targets[0].copy_url %>">
```
Comment (mwest1066) [inline apps/prairielearn/src/pages/partials/question.ejs:221]: Are we guaranteed that `question_copy_targets` isn't the empty list? I see that we check it's not `undefined` above, but not that it's length is more than zero.

**Review by mwest1066 — APPROVED** [review]

File: apps/prairielearn/src/pages/partials/question.ejs (hunk 1 above)
Comment (nwalters512) [reply apps/prairielearn/src/pages/partials/question.ejs:221]: Given that a lot of tests failed, you're 100% right that this is an issue 🙂 I'll fix it!

Comment (nwalters512) [reply apps/prairielearn/src/pages/partials/question.ejs]: This should be addressed by e491d7b.

**Review by jonatanschroeder — COMMENTED** [review]
For completeness, would it make sense to mark the first option as "selected"? I know browsers will typically select the first one by default if none is tagged, but the [W3 specification](https://www.w3.org/TR/2011/WD-html5-20110405/the-button-element.html#the-select-element) doesn't seem to indicate that as required.

**Comment by nwalters512** [comment]
@jonatanschroeder I made that change in c8ded1d.

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**